discord-roles.md

```diff
@@ -0,0 +1,71 @@
+| Server Manager         | PIM                               | Has Manage Server/Admin permission                                |        :x:         |        None         |                None                |
+| Role Manager           | PIM                               | Has Manage Roles permission                                       |        :x:         |        None         |                None                |
+| Channel Manager        | PIM                               | Has Manage Channels permission                                    |        :x:         |        None         |                None                |
+| Announcer              | PIM                               | Has Write Message permission in #announcements/#dev-announcements |        :x:         |        None         |                None                |
+| Steering Council       | SC Members                        | Steering Council role, as we currently have it.                   | :white_check_mark: |        None         |                None                |
+| Moderators             | Moderators                        | Has moderation permissions, i.e. mute, kick, ban, etc.            | :white_check_mark: |        None         |                None                |
+| Maintainers            | Maintainers                       | Has GitHub permissions.                                           | :white_check_mark: |        None         |                None                |
+| Foundation Team        | Foundation Members                | Involved in the NeoForged legal entity.                           | :white_check_mark: |        None         |                None                |
+| Infrastructure Team    | Infrastructure Managers           | Involved in the project infrastructure in some way.               | :white_check_mark: |        None         |                None                |
+| Docs Team              | Documentation writers             | People with permissions on the docs repo.                         | :white_check_mark: |        None         |                None                |
+| Bot Team               | Bot managers                      | People who can manage the bot.                                    | :white_check_mark: |        None         |                None                |
+| NeoForged Team         | All team members                  | Catch-all for all the hoisted roles above.                        |        :x:         |        None         |                None                |
+| Automation             | Camelot                           | Contains camelot so that it can moderate the public.              |        :x:         |        Gray         |                Gear                |
+| Mojang Team            | Mojangsters                       | Mojang people.                                                    |        :x:         |     Mojang Red      |                None                |
```

Comment:
I would dislike calling the role as "Mojang Team", since this too much implies that it is a subteam of the NeoForged Team. Perhaps "Mojangsters" itself suffices, or the endearment "Mojangstas".